experiments/EclipseExample/checkForDep.sh

```diff
@@ -0,0 +1,33 @@
+#this file is used to recompile the sample project and generate sootTestOutput file and call the OneConfigurationRunner
+#"calc-orig-order-1" will have to be modified depending on the test case being used and its contents will have to be modified 
+#only run tests from CalculatorTestCase1, CalculatorTestCase2, or CalculatorTestCase3
```

Comment:
This comment should say directly what needs to be invoked on the command line to use this script and what the expected output is.